docs/detections/api/exceptions/api-summary-exception-container.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[exceptions-api-summary-exception-container]]
+=== Summary exception container
+
+Retrieves an exception container summary.
+
+NOTE: The {kib} Console supports only Elasticsearch APIs. You cannot interact with the {kib} APIs with the Console and must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
```

Comment:
```suggestion
NOTE: Console supports only Elasticsearch APIs. You cannot interact with {kib} APIs with Console and must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
```

Reply:
Checked the Kibana docs and it looks like they're using Console as a proper noun, which is why they don't use an article before it.